src/checkstyle/checkstyle.xml

```diff
@@ -11,6 +11,10 @@
         <property name="headerFile" value="${checkstyle.header.file}"/>
         <property name="fileExtensions" value="java"/>
     </module>
+    <module name="SuppressionFilter">
+        <property name="file" value="src/checkstyle/checkstyle-suppressions.xml"/>
```

Comment:
I think you need to consider to use this property in the Maven Checkstyle plugin configuration in the rot pom instead:
```
 <parameter>
          <name>suppressionsLocation</name>
          <type>java.lang.String</type>
          <since>2.0-beta-2</since>
          <required>false</required>
          <editable>true</editable>
          <description>&lt;p&gt;
Specifies the location of the suppressions XML file to use.
&lt;p&gt;
This parameter is resolved as resource, URL, then file. If successfully
resolved, the contents of the suppressions XML is copied into the
&lt;code&gt;${project.build.directory}/checkstyle-suppressions.xml&lt;/code&gt; file
before being passed to Checkstyle for loading.
&lt;p&gt;
See &lt;code&gt;suppressionsFileExpression&lt;/code&gt; for the property that will
be made available to your Checkstyle configuration.</description>
        </parameter>
```

I mean the point is the we are aiming to remove our `checkstyle.xml` config altogether in favor of that one provided by Spring Cloud Build.

Reply:
It is in Spring Cloud Build's  `checkstyle.xml` now: https://github.com/spring-cloud/spring-cloud-build/pull/79/files

Reply:
Oh! That's is not going to work I guess, but we indeed can give it a try.
The problem that `checkstyle.xml` in our project is picked up from classpath and I'm fully not sure how that relative path to `/src` is going to work...

Reply:
We should test that. Hopefully, it will work.